pieces.py

```diff
@@ -139,7 +182,9 @@ def serialize(self, element):
         if self.author is not None:
             etree.SubElement(m, 'author').text = self.author
         if self.description is not None:
-            etree.SubElement(m, 'description').text = self.description
+            desc = etree.SubElement(m, 'description')
```

Comment:
I would prefer the serialization being uniform regardless whether there are multiple paragraphs or only a single one.